docs/fast_data/monitoring/dashboards/single_views.md

```diff
@@ -4,24 +4,39 @@ title: Single Views
 sidebar_label: Single Views
 ---
 
-This Dashboard is mainly concerned with Single Views related concepts, and a Single View Creator service is required for exposing the metrics of interest.
+This Dashboard is mainly concerned with Single Views related concepts and as such, it needs the Single View Creator and/or Single View Trigger Generator to expose the metrics it displays.
+
+:::caution
+Make sure the services you need the metrics from have the "Enable metric gathering" flag enabled in the console.
+:::
 
 Click <a download target="_blank" href="/docs_files_to_download/single-view-creator.json">**here**</a> to download the dashboard.
 
+### Single View Trigger Generator Service
+
+The Single View Trigger Generator Service is one of the core services of the Fast Data which replaces the Real Time Updater on the strategy execution side of things. 
```

Comment:
let's just link the documentation of the Single View Trigger, I would not write that it replaces the RTU. If you want to say that it does the strategies it's ok, but that's it

Reply:
ok, I'll refrease it

Reply:
Done, wdyt?